pkg/deployment/deployment_item.go

```diff
@@ -34,6 +34,7 @@ type DeploymentItem struct {
 
 	// These values come from the metadata of the kustomization.yml
 	Barrier       bool
+	Message       *string
```

Comment:
No need to have this one here, as the message can only originate from a deployment.yaml and not from a kustomization.yaml.

Reply:
Removed the property